susemanager-sync-data/additional_products.json

```diff
@@ -1,4 +1,191 @@
 [
+    {
```

Comment:
Unless @mcalmer can already provide the right IDs, I'd say this needs to go away?